src/_test/logging.test.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import test from 'ava'
+import * as sinon from 'sinon'
+import { Logger, logger } from '../logging'
+
+test.serial('the custom logger is used when calling each log method', t => {
```

Comment:
Should there be a test that verifies that the default logger is used?

Reply:
Add the test code to check whether the default logger is used or not at 597e523.